File: app/src/main/java/bank/jaeyoung/com/bank/DescendingSort.java
package bank.jaeyoung.com.bank;


import bank.jaeyoung.com.bank.data.DetailTransaction;

import java.util.Comparator;

/**
 * Created by jaeyoung lee on 2018. 10. 17..
 */
public class DescendingSort implements Comparator<DetailTransaction> {
	@Override public int compare(DetailTransaction o1, DetailTransaction o2) {
		return o2.getDate().compareTo(o1.getDate());
	}
}

File: app/src/main/java/bank/jaeyoung/com/bank/DescendingSort.kt
package bank.jaeyoung.com.bank


import bank.jaeyoung.com.bank.data.DetailTransaction
import java.util.*

/**
 * Created by jaeyoung lee on 2018. 10. 17..
 */
class DescendingSort : Comparator<DetailTransaction> {
    override fun compare(o1: DetailTransaction, o2: DetailTransaction): Int {
        return o2.date!!.compareTo(o1.date!!)
    }
}
